ui/src/map/layers/alignment/alignment-linking-layer.ts

```diff
@@ -832,7 +832,7 @@ async function getLinkingData(
             getLinkPointsWithAddresses(state.layoutAlignmentType, state.layoutAlignmentId, {
                 layoutStart: state.layoutAlignmentInterval.start,
                 layoutEnd: state.layoutAlignmentInterval.end,
-                layoutHighlight: expectDefined(first(highlightedItems.layoutLinkPoints)),
+                layoutHighlight: first(highlightedItems.layoutLinkPoints),
```

Comment:
Näissä ongelmana oli tämä `expectDefined()` joka sai tämän hakupromisen failaamaan kun se söikin `undefined`:in. Tämän ongelman löytäminen oli todella hankalaa (meni tunteja paikallistaa ongelma,) joten voisi ehkä harkita jonkinlaista loggausta tms. siellä promisen suorittajassa